keras/callbacks.py

```diff
@@ -2733,6 +2776,14 @@ def on_train_batch_end(self, batch, logs=None):
                 1.0 / batch_run_time,
                 step=self._train_step,
             )
+
+        # `logs` is a `tf.distribute.experimental.coordinator.RemoteValue` when
+        # using asynchronous strategies, for now we just disable `update_freq`
```

Comment:
Let's make this clearer - RemoteValue is particular to ParameterServerStrategy, and not any asynchronous strategies